_conferences/usenixsec2023/results.md

```diff
@@ -27,6 +27,123 @@ artifacts:
 #* 27 Artifact Functional
 #* 20 Results Reproduced

```

Comment:
Can we generate similar text below as a summary. Also we should mention that these are the results from the 1st cycle and do not include the results of the 2nd and 3rd cycle.